daemon/vpn/nordlynx/libtelio/libtelio.go

```diff
@@ -143,35 +143,29 @@ type persistentKeepAliveConfig struct {
 	Stun     int `json:"stun,omitempty"`
 }
 
-func newTelioFeatures() *telioFeatures {
-	return &telioFeatures{
-		Lana: &lanaConfig{},
-		Nurse: &nurseConfig{
-			Qos: &qosConfig{},
-		},
-		Direct: &directConfig{},
-		Derp:   &derpConfig{},
-		Wireguard: &wireguardConfig{
-			PersistentKeepAlive: &persistentKeepAliveConfig{},
-		},
-	}
-}
-
 func handleTelioConfig(eventPath, deviceID, version string, prod bool, remoteConfig remote.RemoteConfigGetter) ([]byte, error) {
-	telioConfig := newTelioFeatures()
+	telioConfig := &telioFeatures{}
 	cfgString, err := remoteConfig.GetTelioConfig(version)
 	if err != nil {
-		log.Printf("getting telio remote config json string: %s\n", err)
+		return nil, fmt.Errorf("getting telio remote config json string: %s\n", err)
 	} else {
 		err := json.Unmarshal([]byte(cfgString), &telioConfig)
 		if err != nil {
-			log.Printf("unmarshaling telio remote config json string: %s\n", err)
-			telioConfig = newTelioFeatures()
+			return nil, fmt.Errorf("unmarshaling telio remote config json string: %s\n", err)
 		}
 	}
-	telioConfig.Lana.EventPath = eventPath
-	telioConfig.Lana.Prod = prod
-	telioConfig.Nurse.Fingerprint = deviceID
+	if telioConfig.Lana != nil {
+		telioConfig.Lana.EventPath = eventPath
+		telioConfig.Lana.Prod = prod
+		if telioConfig.Nurse != nil { // nurse depends on lana
```

Comment:
This is minor, but it seems to me that both `lana` is dependent on `nurse` and vice-versa. So I'd either write this 
```
if telioConfig.Lana != nil {
<set lana stuff>
} 
if telioConfig.Nurse != nil {
<set nurse stuff>
}
```
or a safer approach:
```
if telioConfig.Lana != nil && telioConfig.Nurse != nil {
 <set lana and nurse stuff>
} else {
 telioConfig.Lana = nil
 telioConfig.Nurse = nil
}
```

Reply:
In `telio` doc I find only `Nurse` being dependent on `Lana`, but not vice versa.